app/controllers/insured/plan_shoppings_controller.rb

```diff
@@ -125,6 +126,15 @@ def print_waiver
     @hbx_enrollment = HbxEnrollment.find(params.require(:id))
   end
 
+  def employee_waiver_notice(person)
+    begin
+      census_employee = person.employee_roles.first.census_employee
```

Comment:
Person can have multiple employee roles. You have to pick the correct employee_role.